src/Storage/functions.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
I'd much prefer if we could provide methods on `StorageClient` to do this, rather than namespaced functions. Namespaced functions haven't been used until now, and I don't see a really strong case for adding them.